pydantic/errors.py

```diff
@@ -101,6 +112,9 @@ def __init__(self, **ctx: Any) -> None:
     def __str__(self) -> str:
         return self.msg_template.format(**self.__dict__)
 
+    def __reduce__(self):  # type: ignore
```

Comment:
@samuelcolvin I thought ignore the typing made more sense as it made code more complicated for nothing.
But here is the commit to add types https://github.com/PrettyWood/pydantic/commit/753abddfefd59d5d8eda7bbdeaef48d973157d36
If you want me to, I'll add it to this PR

Reply:
I think we should include type hints if at all possible